engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -62,6 +62,17 @@
 	 */
 	String ALLOW_PARALLEL_METHODS_DEFINE_PARAMETER_CONSTRAINTS = "hibernate.validator.allow_parallel_method_parameter_constraint";
 
+	/**
```

Comment:
Did you keep the docs on purpose? I'd say let's remove it.

Reply:
Doc removed.